doc/usage/extensions/autodoc.rst

```diff
@@ -325,7 +325,16 @@ inserting them into the page source under a suitable :rst:dir:`py:module`,
 
    By default, without ``annotation`` option, Sphinx tries to obtain the value of
    the variable and print it after the name.
+   
+   The ``novalue`` option can be used instead of a blank ``annotation`` to show the
```

Comment:
I guess ``:novalue:`` should be mentioned in a ``.. versionchanged::`` at the end of this section.